sys-kernel/dracut-crypt-ssh/dracut-crypt-ssh-1.0.6.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="early unlocking of encrypted systems via ssh for dracut"
```

Comment:
Please fix the variable order and spacing to match the skel. Also, where did you get the template for this ebuild? I'm noticing a repeating misordering and I can't locate the source.

```
DESCRIPTION
HOMEPAGE
SRC_URI

LICENSE
SLOT
KEYWORDS
IUSE

DEPEND
RDEPEND
```

Reply:
I started with the skeleton that automatically appears in in vim.

I guess the LICENSE line wandered up after I corrected the license from "gpl-2" to "GPL-2" before I opened the pull request, so no common source for that error, I think.